src/components/IfOperand/index.jsx

```diff
@@ -18,6 +20,13 @@ const IfOperand = ({ words = [], onDelete }) => {
       <Grid item xs={12} sm={3} md={3}>
```

Comment:
I put this component in replacement of the `WordPropsSelector`, so you can remove it. So that we have
WordSelector |  WordPropsSelector | OperationSelector | Value